actors/miner/src/lib.rs

```diff
@@ -901,7 +901,10 @@ impl Actor {
                 continue;
             }
 
-            sector_numbers.set(update.sector_number);
+            if sector_numbers.try_set(update.sector_number).is_err() {
+                info!("invalid sector number, skipping");
```

Comment:
I think aborting is appropriate here. This is essentially malformed input.

Reply:
I agree, but the rest of this code ignores malformed input (e.g., take a look at the window post deadline check later).

Reply:
https://github.com/filecoin-project/builtin-actors/issues/169